src/TConsole.cpp

```diff
@@ -546,6 +552,9 @@ TConsole::TConsole(Host* pH, const QString& name, const ConsoleType type, QWidge
         mpCommandLine->adjustHeight();
     }
 
+    // Create Announcer for screen reader support
+    new Announcer(this);
```

Comment:
You can use the existing `mudlet::announce()` to announce (if needed)

Reply:
removed